docs/quickstart.md

```
@@ -83,7 +85,7 @@ with Document.open("contract.docx") as doc:
     doc.save()
 ```
 
-This creates a tracked deletion of `30 days` and a tracked insertion of `60 days`.
+This creates a tracked deletion of `30` and a tracked insertion of `60`. `replace()` trims the common prefix and suffix and tracks only the differing words, so the unchanged surrounding word (` days`) is left intact rather than deleted and re-inserted.
```

markdownlint-cli2 (0.23.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)